lib/improver/tests/orographic_enhancement/test_OrographicEnhancement.py

```diff
@@ -631,88 +631,69 @@ def setUp(self):
-        expected_data = np.array(
-            [[np.nan, 2.777778e-07, 3.8888888e-07, np.nan],
-             [np.nan, np.nan, np.nan, np.nan]], dtype=np.float32)
+        """Test the cube is changed only in units (to m s-1)"""
         original_converted = 2.7777778e-07 * self.orogenh
-        expected_mask = np.where(np.isfinite(expected_data), False, True)
-        output, regridded_output = self.plugin._create_output_cubes(
+
+        output = self.plugin._create_output_cube(
             self.orogenh, self.temperature)
         self.assertArrayAlmostEqual(output.data, original_converted)
-        self.assertTrue(np.allclose(regridded_output.data.data,
-                                    expected_data, equal_nan=True))
-        self.assertArrayEqual(regridded_output.data.mask, expected_mask)
 
     def test_metadata(self):
-        """Check output metadata on both cubes is as expected"""
+        """Check output metadata on cube is as expected"""
         hi_res_attributes = self.temperature.attributes
         for key, val in self.plugin.topography.attributes.items():
             hi_res_attributes[key] = val
 
-        tref = sort_coord_in_cube(
-            self.temperature, self.temperature.coord(axis='y'))
-        output, regridded_output = self.plugin._create_output_cubes(
-            self.orogenh, self.temperature)
+        output = self.plugin._create_output_cube(self.orogenh,
+                                                 self.temperature)
 
         for axis in ['x', 'y']:
             self.assertEqual(output.coord(axis=axis),
                              self.plugin.topography.coord(axis=axis))
-            self.assertEqual(regridded_output.coord(axis=axis),
-                             tref.coord(axis=axis))
-
-        for cube in [output, regridded_output]:
-            self.assertEqual(cube.name(), 'orographic_enhancement')
-            self.assertEqual(cube.units, 'm s-1')
-            for t_coord in ['time', 'forecast_period',
-                            'forecast_reference_time']:
-                self.assertEqual(
-                    cube.coord(t_coord), self.temperature.coord(t_coord))
-        self.assertDictEqual(
-            regridded_output.attributes, self.temperature.attributes)
+
+        self.assertEqual(output.name(), 'orographic_enhancement')
+        self.assertEqual(output.units, 'm s-1')
+        for t_coord in ['time', 'forecast_period', 'forecast_reference_time']:
+            self.assertEqual(
+                output.coord(t_coord), self.temperature.coord(t_coord))
         self.assertDictEqual(output.attributes, hi_res_attributes)
 
     def test_grid_metadata(self):
         """Test specific grid and model metadata inheritance"""
-        output, regridded_output = self.plugin._create_output_cubes(
-            self.orogenh, self.temperature)
+        output = self.plugin._create_output_cube(self.orogenh,
+                                                 self.temperature)
```

Comment:
This is just a comment, but it's an informal local convention that we tend to prefer newline indenting over lining up the arguments in IMPROVER.  Ie:
```
output = self.plugin._create_output_cube(
    self.orogenh, self.temperature)
```